File: Geek Assistant (Android AIO)/Forms/Extra/Preparing.vb
Public Class Preparing

    'Private Sub Preparing_Load(sender As Object, e As EventArgs) Handles MyBase.

    'End Sub
    Private WithEvents CheckADBProcess_Timer As New Timer With {.Interval = 100}
    Private Sub Preparing_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        GA_SetTheme.Run(Name, True)
        Preparing_Label.Text = txt_RandomWorkingText()
        CheckADBProcess_Timer.Enabled = True
    End Sub
    Private Sub CheckADBProcess_Timer_Tick() Handles CheckADBProcess_Timer.Tick
        If Process.GetProcessesByName("adb").Count > 0 Then
            CheckADBProcess_Timer.Enabled = False
            Main.Show()
            Close()
        End If
    End Sub
End Class

File: Geek Assistant (Android AIO)/Forms/Extra/Preparing.cs
using System;
using System.Diagnostics;
using System.Linq;
using System.Windows.Forms;

namespace GeekAssistant.Forms {
    public partial class Preparing : Form {
        public Preparing() {
            InitializeComponent();
        }
        private void AssignEvents() {
            CheckADBProcess_Timer.Tick += CheckADBProcess_Timer_Tick;
        }
        private Timer CheckADBProcess_Timer = new() { Interval = 100 };
        private void Preparing_Load(object sender, EventArgs e) {
            AssignEvents();
            GA_SetTheme.Run(Name, true);
            Preparing_Label.Text = txt.RandomWorkText();
            CheckADBProcess_Timer.Enabled = true;
        }
        private void CheckADBProcess_Timer_Tick(object sender, EventArgs e) {
            if (Process.GetProcessesByName("adb").Count() > 0) {
                CheckADBProcess_Timer.Enabled = false;
                common.Home.Show();
                Dispose();
            }
        }
    }
}
